tests/fast-integration/kcp/client.js

```diff
@@ -344,7 +344,7 @@ class KCPWrapper {
       if (err.stderr === undefined) {
         throw new Error(`failed to process kcp binary output: ${err.toString()}`);
       }
-      throw new Error(`kcp command failed: args: ${args} ,${err.stderr.toString()}`);
+      throw new Error(`kcp command failed: ${err.stderr.toString()}`);
```

Comment:
For reviewer:
The `args` contains username and password in raw, which is not safe to be printed to console.